### PR TITLE
document End of Life policy

### PR DESCRIPTION
**Pull request by ndeloof**
Document Compose V1 retirement ("End of Life") as Compose V2 is ready for G.A.
This PR is mostly for review regarding the formalism, actual dates are subject to discussion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,31 @@ Docker Compose
...
+We are currently polishing the next generation of Docker Compose, to be released soon... :stopwatch: 
+- Read more on [RC1 announcement](https://www.docker.com/blog/start-dev-environments-locally-compose-v2-rc-1-and-more-in-docker-desktop-3-6/). 
+- Check the [v2 branch](https://github.com/docker/compose/tree/v2) on this repo.
+
+we are working towards providing an easy way to install Compose V2 on Linux. Once this is available, Compose V2 will be marked as generally available, our current target for this is the end of October.
+
+V1 vs V2 transition :hourglass_flowing_sand:
+--------------------------------------------
+
+"Generally Availability" will mean:
```
Comment (lorenrh) [inline README.md:19]:
```suggestion
"Generally Available" will mean:
```

File: README.md [hunk 2]
```diff
@@ -4,10 +4,31 @@ Docker Compose
...
-| --- |
-| We are currently polishing the next generation of Docker Compose, to be released soon... :stopwatch: <br> - Read more on [RC1 announcement](https://www.docker.com/blog/start-dev-environments-locally-compose-v2-rc-1-and-more-in-docker-desktop-3-6/). <br> - Check the [v2 branch](https://github.com/docker/compose/tree/v2) on this repo. |
+** Compose V2 is on its way! :star_struck: **
+---------------------------------------------
 
+We are currently polishing the next generation of Docker Compose, to be released soon... :stopwatch: 
+- Read more on [RC1 announcement](https://www.docker.com/blog/start-dev-environments-locally-compose-v2-rc-1-and-more-in-docker-desktop-3-6/). 
+- Check the [v2 branch](https://github.com/docker/compose/tree/v2) on this repo.
+
+we are working towards providing an easy way to install Compose V2 on Linux. Once this is available, Compose V2 will be marked as generally available, our current target for this is the end of October.
```
Comment (lorenrh) [inline README.md:14]:
```suggestion
We are working towards providing an easy way to install Compose V2 on Linux. Once this is available, Compose V2 will be marked as generally available, our current target for this is the end of October.
```

File: README.md [hunk 3]
```diff
@@ -4,10 +4,31 @@ Docker Compose
...
+
+V1 vs V2 transition :hourglass_flowing_sand:
+--------------------------------------------
+
+"Generally Availability" will mean:
+- New features and bug fixes will only be considered in the V2 codebase 
+- Users on Mac/Windows will be defaulted into Docker Compose V2, but can still opt out through the UI and the cli. This means when running `docker-compose` you will actually be running `docker compose`
```
Comment (lorenrh) [inline README.md:21]:
```suggestion
- Users on Mac/Windows will be defaulted into Docker Compose V2, but can still opt out through the UI and the CLI. This means when running `docker-compose` you will actually be running `docker compose`
```

File: README.md [hunk 4]
```diff
@@ -4,10 +4,31 @@ Docker Compose
...
+- Our goal currently is for users on Linux to receive Compose v2 with the latest version of the docker CLI, but is pending some technical discussion. Users will be able to use [compose switch](https://github.com/docker/compose-switch) to enable redirection of `docker-compose` to `docker compose`
```
Comment (lorenrh) [inline README.md:22]:
```suggestion
- Our current goal is for users on Linux to receive Compose v2 with the latest version of the docker CLI, but is pending some technical discussion. Users will be able to use [compose switch](https://github.com/docker/compose-switch) to enable redirection of `docker-compose` to `docker compose`
```

File: README.md [hunk 5]
```diff
@@ -4,10 +4,31 @@ Docker Compose
...
+- Docker Compose V1 will continue to be maintained regarding security issues
+- [v2 branch](https://github.com/docker/compose/tree/v2) will become the default one at that time
+
+
```
Comment (lorenrh) [inline README.md:26]: extra line

File: README.md [hunk 6]
```diff
@@ -4,10 +4,29 @@ Docker Compose
...
+We are currently polishing the next generation of Docker Compose, to be released soon... :stopwatch: 
+- Read more on [RC1 announcement](https://www.docker.com/blog/start-dev-environments-locally-compose-v2-rc-1-and-more-in-docker-desktop-3-6/). 
+- Check the [v2 branch](https://github.com/docker/compose/tree/v2) on this repo.
+
+We are working towards providing an easy way to install Compose V2 on Linux. Once this is available, Compose V2 will be marked as generally available, our current target for this is the end of October.
+
+V1 vs V2 transition :hourglass_flowing_sand:
+--------------------------------------------
+
+"Generally Availe" will mean:
```
Comment (stephanierifai) [inline README.md:19]:
```Suggestion
"Generally Available" will mean:
```

**Review by stephanierifai — COMMENTED** [review]
This LGTM, minor nit for "available"

**Pull request merged**